prov/efa/src/rxr/rxr_op_entry.c

```diff
@@ -258,7 +258,11 @@ void rxr_tx_entry_set_max_req_data_size(struct rxr_ep *ep, struct rxr_tx_entry *
 
 	num_req = (mulreq_total_data_size - 1)/max_req_data_capacity + 1;
 
-	tx_entry->max_req_data_size = ofi_get_aligned_size((mulreq_total_data_size - 1)/num_req + 1, MEMORY_ALIGNMENT);
+	if (efa_mr_is_cuda(tx_entry->desc[0])) {
```

Comment:
Was it a bug before as well?

Reply:
Yes. It only impact performance though, will not cause hang.